Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.cpp

```diff
@@ -872,6 +877,26 @@ void MediaPlayerPrivateGStreamerMSE::emitPlayReadySession(PlayreadySession* sess
 #endif
 #endif
 
+#if (ENABLE(LEGACY_ENCRYPTED_MEDIA) || ENABLE(LEGACY_ENCRYPTED_MEDIA_V1)) && USE(OCDM)
+void MediaPlayerPrivateGStreamerMSE::emitOpenCDMSession()
+{
+    if (!m_cdmSession)
+        return;
+
+    CDMSessionOpenCDM* cdmSession = getOpenCDMSession();
```

Comment:
This function shouldn't exist, right? We agreed on renaming it to openCDMSession.